phpBB/language/en/acp/attachments.php

```diff
@@ -117,7 +117,7 @@
 
 	'IMAGE_LINK_SIZE'			=> 'Image link dimensions',
 	'IMAGE_LINK_SIZE_EXPLAIN'	=> 'Display image attachment as an inline text link if image is larger than this. To disable this behaviour, set the values to 0px by 0px.',
-	'IMAGICK_PATH'				=> 'Imagemagick path',
+	'IMAGICK_PATH'				=> 'ImageMagick path',
 	'IMAGICK_PATH_EXPLAIN'		=> 'Full path to the imagemagick convert application, e.g. <samp>/usr/bin/</samp>.',
```

Comment:
This still says imagemagick. I guess that's based on the name of the executable?